sign/io-sign.js

```diff
@@ -1361,7 +1369,7 @@ let A = class extends f {
   connectedCallback() {
     if (super.connectedCallback(), document.getElementById("io-sign-theme-css") === null) {
       const e = document.createElement("link");
-      e.id = "io-sign-theme-css", e.rel = "stylesheet", e.href = "https://stlucadev.z6.web.core.windows.net/sdk/theme.css", e.addEventListener("load", () => {
+      e.id = "io-sign-theme-css", e.rel = "stylesheet", e.href = "https://assets.cdn.io.pagopa.it/sign/theme.css", e.addEventListener("load", () => {
```

Comment:
Given that this is the only relevant change, excluding build changes, and also that the `redirectOrShowQrCode` function has been fixed in 72c348a0ae23c9c418ab10b37acfd42ab18d84e0, I'd say this LGTM.